packages/theme/components/AppFooter.vue

```diff
@@ -82,6 +82,13 @@ export default {
 .footer {
   margin-bottom: 3.75rem;
 
+  .sf-link:hover {
+    .sf-menu-item__label{
+      color: white !important;
```

Comment:
```suggestion
      color: var(--c-white) !important;
```
dont use hardcoded colors